provider/ansible/request.rb

```diff
@@ -72,38 +50,18 @@ def properties_with_classes(properties)
 
       private
 
-      def request_property(prop, hash_name, module_name, indent_width)
-        indent(
-          ["#{unicode_string(prop.api_name)}: #{request_output(prop, hash_name, module_name)}"],
-          indent_width
-        )
-      end
-
-      def response_property(prop, hash_name, module_name, indent_width)
-        indent(
-          ["#{unicode_string(prop.api_name)}: #{response_output(prop, hash_name, module_name)}"],
-          indent_width
-        )
-      end
-
+      # This is outputting code and code is easier to read on one line.
```

Comment:
I don't like situations where we do string manipulation to write Python code because it makes the Python code hard to read. But making these things multi-line to satisfy Rubocop makes them ridiculously hard to read for a bad reason.